Servers/swagger.yaml

```
@@ -1,12 +1,127 @@
 openapi: 3.0.0
 info:
-  title: Compliance Management API
-  description: API documentation for Verifywise AI compliance management application
+  title: VerifyWise API
+  description: |
+    ## Overview
+    The VerifyWise API enables programmatic access to AI compliance management features including project management, risk assessments, vendor management, and compliance framework implementations (EU AI Act, ISO 42001, ISO 27001).
+
+    ## Getting Started
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)